content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -235,15 +237,15 @@ Install Istio and expose the control plane so that your virtual machine can acce
       namespace: "${VM_NAMESPACE}"
     spec:
       metadata:
-        labels:
-          app: "${VM_APP}"
-      template:
-        serviceAccount: "${SERVICE_ACCOUNT}"
-        network: "${NETWORK}"
+            labels:
+              app: "${VM_APP}"
+          template:
+            serviceAccount: "${SERVICE_ACCOUNT}"
+            network: "${VM_NETWORK}"
     EOF
     {{< /text >}}
 
-    1. Push the `WorkloadGroup` to the cluster:
+1. Push the `WorkloadGroup` to the cluster:
```

Comment:
This one is still not rendering correctly as item `2.`. Try indenting the list items 4 spaces and the text sections 8 spaces. If that doesn't work, maybe nested list items don't work in which case I would suggest just making them straight text (remove the numbering), since there are only 2 steps here anyway.

Reply:
I've just removed the numbering. Looks good now, thanks for the tip!